### ISSUE TITLE
Pre-trained model

### ISSUE THREAD

**Issue by Bedrettin-Cetinkaya** [fxia22/pointnet.pytorch#37]
Is it possible to share pre-trained model?
Thanks.

**Comment by zeinaabuaisheh** [comment]
Hello, I also have the same request, can I please get the pretrained model?
Thanks a lot

**Comment by JackeyDeng** [comment]
yes, I want the pretrained model too.

**Comment by Arnoldnuo** [comment]
I want the model too, thank you.

**Comment by YiruS** [comment]
Same here. Would very much appreciate it if we could have the pretrained model.

**Comment by albertotono** [comment]
I was able to train, and even if display a final accuracy 0.9787752261656228, when I tested on random sample, it get right 5/15 more or less. So it would be great to have a good model if available.

**Comment by SagarAChitnis** [comment]
Hello, your work is good. In order to verify the results, can you please add your pre-trained model for Modelnet or Shapenet? Thank you

**Comment by Hasindri** [comment]
If you have the checkpoint for pointnet classification, can you share? When I train the network by myself, I get a low accuracy
